contracts/strategies/kyber/strategy-ky-avax-knc.sol

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: MIT	
+pragma solidity ^0.6.7;
+
+import "../bases/strategy-kyber-farm-base.sol";
+
+/// @notice This is the strategy contract for Kyber's AVAX-KNC pair with KNC and AVAX rewards
+contract StrategyKyAvaxKnc is StrategyKyberFarm {
+    address public avax_knc = 0x4797D61A80FBb90Aa060c6d8Fe9991c41C4868BF;
+    address public avax_knc_gauge = 0x845d1D0D9b344fbA8a205461B9E94aEfe258B918;
+
+    uint256 public avax_knc_poolId = 3; 
+    uint256 public index; 
+
+    constructor(
+        address _governance,
+        address _strategist,
+        address _controller,
+        address _timelock
+    )
+    public 
+    StrategyKyberFarm(
+        avax_knc_gauge,
+        avax_knc_poolId,
+        avax_knc, 
+        _governance, 
+        _strategist, 
+        _controller, 
+        _timelock
+    )
+    {
+        index = 0;
+    } 
+    
+    // ***** Collects Rewards and Restakes ****** // 
+    function harvest() public override onlyBenevolent {
+
+        // Harvests the rewards from the specific pool on kyber finance 
+        IKyber(avax_knc_gauge).harvest(avax_knc_poolId);
+
+        // Retrieve reward from the vesting contract and increases index for the next reward 
+        uint256[] memory indices = new uint256[](1);
+        indices[0] = index;
+        IRewardLocker(vesting).vestScheduleAtIndices(knc, indices);
+        index = index + 1;
+         
+
+
+        // Take Avax Rewards    
```

Comment:
I'd expand this comment to add that we're swapping KNC to WAVAX, taking our fee, then swapping half WAVAX back to KNC